src/__test__/index.test.ts

```diff
@@ -0,0 +1,8 @@
+import { debug as cdebug } from 'console';
+
+describe('index', () => {
```

Comment:
try to not to commit stubs

Reply:
Thank you for pointing out. It was just the first useless test I used to develop the pipelines